docs/Systems/Tailwind/GettingStarted.stories.mdx

```diff
@@ -0,0 +1,124 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { CATEGORIES } from "../../../storybook/constants"
+import TailwindPlay from "./assets/tailwind-play.gif"
+import CssGeneration from "./assets/css-generation.png"
+import InlineFold from "./assets/inline-fold.gif"
+
+<Meta title={`${CATEGORIES.tailwind}/Getting Started`} />
+
+# Getting Started  
+
+This page describes the steps needed to use our [Kaizen Tailwind preset](https://github.com/cultureamp/kaizen-design-system/tree/main/packages/tailwind), not Tailwind itself.  
+To learn more about Tailwind and how to install it in your project, check out there docs [here](https://tailwindcss.com/docs/installation).
+
+- [1. Install the preset](#1-install-the-preset)
+- [2. Implement the preset](#2-implement-the-preset)
+- [3. Add the Tailwind directives](#3-add-the-tailwind-directives)
+- [4. Useful tools](#4-useful-tools)
+
+Note: Projects created from [frontend-template](https://github.com/cultureamp/frontend-template) should have steps 1 - 3 completed out of the box.  
+If your project was created from this template, you should only need to do [Step 4: Useful tools](#4-useful-tools).
```

Comment:
Move this above the list and I think we're good to go

Reply:
Done 👍